config/assets/default.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
         'public/lib/angular-ui-router/release/angular-ui-router.js',
         'public/lib/angular-ui-utils/ui-utils.js',
         'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
```

Comment:
Having applied this, the problem is here: you also need and it has to be first `'public/lib/angular-bootstrap/ui-bootstrap.js'`